### PR TITLE
Workflow Automated Test - LRQA-68840 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -333,12 +333,44 @@ definition {
...
+			workflowTask = "Review");
 
-		// TODO LRQA-68840 CanOrderAssignedtoMyRolesTasksByDueDate pending implementation
+		Notifications.goToFilterValues(filterValues = "Due Date");
 
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry 2",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_DUE_DATE",
+			value1 = "Within 8 Months");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:373]: Try to create macros from these functions when you feel the need to use them in `.testcase `files. Complementing the  comment above, ideal is to use a macro to ensure the position of items, you can replace this piece and use the same _macro_ `viewMyWorkflowTaskItemPosition` and _path_ `MY_WORKFLOW_TASK_ITEM_POSITION` that you have created at PR #843. Thank you!

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Review by lucasperj — APPROVED** [review]

**Review by lucasperj — APPROVED** [review]
Very good! Thanks

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3)

**Pull request closed without merging**